.github/workflows/build.yaml

```diff
@@ -83,7 +83,7 @@ jobs:
     docker_builds:
       runs-on: ubuntu-latest
       needs: tarball
-      name: Docker Build (${{ matrix.target }})
```

Comment:
I think (?) this name is load bearing in terms of the 'required' checks for GitHub actions. It doesn't seem to be effecting this PR so it's probably okay but we may need to tweak the settings after this lands